torch/testing/_internal/common_nn.py

```diff
@@ -4837,7 +4864,7 @@ def __call__(self, test_case):
             # TODO: do this with in-memory files as soon as torch.save will support it
             with TemporaryFile() as f:
                 test_case._forward(module, input)
-                torch.save(module, f)
+                torch.save(module, f)  # type: ignore[arg-type]
```

Comment:
Should also not be ignored, instead fix the `torch.save` annotation by adding `IO[bytes]` - see https://stackoverflow.com/questions/59693359/when-to-use-iostr-iobytes-and-textio-binaryio-in-python-type-hinting